Typescript errors in wonka 6.3.3

Building my project works correctly with wonka v6.3.2, but not with v6.3.3.  The issue is that typescript finds a issues in wonka.d.ts 

> node_modules/wonka/dist/wonka.d.ts:1323:13 - error TS2339: Property 'observable' does not exist on type 'SymbolConstructor'.
> 
> 1323     [Symbol.observable]?(): Observable<T>;
>                  ~~~~~~~~~~
>                  
> node_modules/wonka/dist/wonka.d.ts:1352:13 - error TS2339: Property 'observable' does not exist on type 'SymbolConstructor'.
> 
> 1352     [Symbol.observable](): Observable<T>;
>                  ~~~~~~~~~~

This can be resolved by adding the following block to the wonka.d.ts file

```
declare global {
  interface SymbolConstructor {
    readonly observable: symbol;
  }
}
```

It appears that this was recently moved from the observable.ts file to the helpers.ts file, but I can't tell why it doesn't appear in the finished wonka.d.ts file, but it could be that index.ts doesn't contain a line like

`export * from './helpers';`

I will continue to look into this when I have more time.

--- a/.changeset/fresh-tips-yell.md
+++ b/.changeset/fresh-tips-yell.md
@@ -1,0 +1,5 @@
+---
+'wonka': patch
+---
+
+Add missing `Symbol.observable` global declaration back to typings.

--- a/src/helpers.ts
+++ b/src/helpers.ts
@@ -1,11 +1,5 @@
 import { TalkbackFn, TeardownFn, Start, Push, SignalKind } from './types';
 
-declare global {
-  interface SymbolConstructor {
-    readonly observable: symbol;
-  }
-}
-
 /** Placeholder {@link TeardownFn | teardown functions} that's a no-op.
  * @see {@link TeardownFn} for the definition and usage of teardowns.
  * @internal

--- a/src/observable.ts
+++ b/src/observable.ts
@@ -1,6 +1,14 @@
 import { Source, SignalKind, TalkbackKind } from './types';
 import { push, start, talkbackPlaceholder, observableSymbol } from './helpers';
 
+// NOTE: This must be placed in an exported file for `rollup-plugin-dts`
+// to include it in output typings files
+declare global {
+  interface SymbolConstructor {
+    readonly observable: symbol;
+  }
+}
+
 /** A definition of the ES Observable Subscription type that is returned by
  * {@link Observable.subscribe}
  *

